src/main/java/org/folio/consortia/service/impl/SharingInstanceServiceImpl.java

```diff
@@ -105,15 +105,40 @@ public SharingInstance start(UUID consortiumId, SharingInstance sharingInstance)
 
       sharingInstance.setStatus(Status.IN_PROGRESS);
     }
-    SharingInstanceEntity savedSharingInstance = sharingInstanceRepository.save(toEntity(sharingInstance));
+
+    SharingInstanceEntity savedSharingInstance = saveSharingInstance(sharingInstance);
     log.info("start:: sharingInstance with id: {}, instanceId: {}, sourceTenantId: {}, targetTenantId: {} has been saved with status: {}",
       savedSharingInstance.getId(), savedSharingInstance.getInstanceId(), sourceTenantId, targetTenantId, savedSharingInstance.getStatus());
     return converter.convert(savedSharingInstance, SharingInstance.class);
   }
 
+  /**
```

Comment:
The comment make sense however please review this version and update if you don't mind:
```
   * This method saves sharing instance record to database.
   * If some record was created during the previous attempt then it will be updated[ with new status]. Otherwise new record will be inserted.
```
[] - optional text

Reply:
Thanks to suggestion, I added some words from your suggestion and improved the text